packages/wasm/package.json

```diff
@@ -5,6 +5,7 @@
   "license": "MIT",
   "type": "module",
   "scripts": {
+    "precompile": "tsx ../../apps/extension/src/utils/download-proving-keys.ts",
```

Comment:
excellent, i like how this leverages the pre/post auto-execute feature of npm. it might be more technically correct as a 'post' script.

we might look at using more of these pre/post scripts for dependency management outside of turbo, which should be aware of them and manage cache appropriately - `playwright-install` before certain tests is another good candidate for this